templates/workspaces/base/porter.yaml

```diff
@@ -126,6 +130,10 @@ parameters:
     type: string
     default: "GRS"
     description: "The redundancy option for the storage account in the workspace: GRS (Geo-Redundant Storage) or ZRS (Zone-Redundant Storage)."
+  - name: auto_grant_workspace_consent
+    type: boolean
+    default: true
+    description: "Whether to automatically grant consent for the workspace app registration."
```

Comment:
Can we reuse working from elsewhere?

Reply:
Not sure I follow, can you clarify please?